packages/gatsby-recipes/src/recipe-machine/index.js

```diff
@@ -148,7 +148,16 @@ const recipeMachine = Machine(
           },
         },
         on: {
-          CONTINUE: `applyingPlan`,
+          CONTINUE: [
+            {
+              target: `applyingPlan`,
+              cond: `hasNoErrors`,
+            },
+            {
+              target: `doneError`,
+              cond: `hasErrors`,
+            },
+          ],
```

Comment:
Technically this isn't necessary, xstate will try the first guard if it's false it goes to the next. They don't run serially or in parallel.
```suggestion
          CONTINUE: [
            {
              target: `doneError`,
              cond: `hasErrors`,
            },
            {
              target: `applyingPlan`,
            },
          ],
```

Reply:
👍 good idea